### ISSUE TITLE
There is a writing error in the paper

### ISSUE THREAD

**Issue by zchuz** [Yui010206/SeViLA#3]
There is a writing error in the paper
The batchsize in the pre-training and refinement phase should be 16 per gpu instead of 64 per gpu.

**Comment by Yui010206** [comment]
For pre-training and self-refinement, we sample 4 frames from each video samples, and calculate gradients on reshaped batch. The batch size on each GPU is 4x16, consistent with the specifications outlined in the paper. Thanks for your suggestion, we will clarify this later.

**Comment by zchuz** [comment]
Thank you for your reply, I did not take into account that each video samples four frames.
